src/main/java/org/spring/FullVersion/FamilyService.java

```diff
@@ -85,7 +85,7 @@ public void deleteAllChildrenOlderThen(int age) {
 
         familyDao.getAllFamilies().stream()
                 .peek(family -> family.getChildren().removeIf(child -> {
-                    LocalDate birthDate = convertToLocalDate(child.describeAge());
+                    LocalDate birthDate = convertToLocalDate(Integer.toString(child.getAge()));
```

Comment:
you already get age from child.getAge() why you parse it to string and send convertToLocalDate?

Reply:
LocalDate birthDate = LocalDate.ofEpochDay(child.getAge()); 
Do you want it in this form? @sabuhihuseynov